app/models/concerns/exportable.rb

```diff
@@ -0,0 +1,15 @@
+module Exportable
+  extend ActiveSupport::Concern
```

Comment:
```
diff --git a/app/models/concerns/exportable.rb b/app/models/concerns/exportable.rb
index 35638c3..1551a40 100644
--- a/app/models/concerns/exportable.rb
+++ b/app/models/concerns/exportable.rb
@@ -1,7 +1,10 @@
 module Exportable
-  extend ActiveSupport::Concern
   require 'csv'

+  def self.included base
+    base.extend ClassMethods
+  end
+
   module ClassMethods
     def as_csv
       CSV.generate do |csv|
```

Only change made to make it PORO. Tests pass. I'm kinda meh on which way to take, maybe lean toward PORO.